components/FlashCard/index.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+import { collections } from "@/lib/data";
+import { useState } from "react";
+
+const StyledCard = styled.li`
+  background-color: ${({ $showAnswer }) =>
+    $showAnswer ? "#93E9BE" : "lightgray"};
+  list-style: none;
+  width: 80%;
+  border-radius: 20px;
+  padding: 25px 25px;
```

Comment:
The second value is redundant since both values are the same.

```suggestion
  padding: 25px;
```